messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ClusterConnectionDefinition.java

```diff
@@ -78,9 +79,9 @@
 
     public static final SimpleAttributeDefinition ALLOW_DIRECT_CONNECTIONS_ONLY = create("allow-direct-connections-only", BOOLEAN)
             .setDefaultValue(new ModelNode(false))
-            .setAllowNull(true)
+            .setRequired(false)
             .setAllowExpression(true)
-            .setAlternatives(CommonAttributes.DISCOVERY_GROUP)
+            .setRequires(STATIC_CONNECTORS)
```

Comment:
if neither CONNECTOR_REFS or DISCOVERY_GROUP_NAME are defined, the add op will fail anyway. I think this change is good.